pages/content/amp-dev/page-experience.html

```diff
@@ -64,7 +83,5 @@ <h1 class="ap-o-stage-content-headline">
       }
     </script>
   </amp-state>
-  {% include '/views/partials/pixi.j2' %}
-
-  {% endif %}
+  {% include '/views/partials/pixi.j2' %} {% endif %}
```

Comment:
nit: can you move the `endif` to the next line

Reply:
Ah, yes thanks! I must have a local setting that keeps auto formatting these.